app/importers/RTMediaMigration.php

```diff
@@ -378,11 +378,11 @@ function test() {
 		} else {
 			?>
 			<div class="error">
-				<p> <?php esc_html_e( 'Please Backup your ', 'buddypress-media' ); ?>
-					<strong><?php esc_html_e( 'DATABASE', 'buddypress-media' ); ?></strong>
-					<?php esc_html_e( ' and ', 'buddypress-media' ); ?>
-					<strong><?php esc_html_e( 'UPLOAD', 'buddypress-media' ); ?></strong>
-					<?php esc_html_e( ' folder before Migration.', 'buddypress-media' ); ?>
+				<p>
+					<?php
+					/* translators: %1$s gets replaced by '<strong>', and %2$s by '</strong>' */
+					echo sprintf( esc_html__( 'Please Backup your %1$sDATABASE%2$s and %1$sUPLOAD%2$s folder before Migration.', 'buddypress-media' ), '<strong>', '</strong>' );
```

Comment:
you can use `printf ` here to avoid excessive processing

Reply:
@milindmore22 Can you share something about `excessive processing`?

Reply:
in simple terms 
`printf` will directly output values 
`sprintf ` will return values and `echo` does output job

so by directly using `printf` to output values, we are saving excessive processing of returning values and printing them.